hles/transformation/src/test/scala/org/broadinstitute/monster/dap/environment/WalkabilityTransformationsSpec.scala

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.monster.dap.dog
+
+import org.broadinstitute.monster.dap.RawRecord
+import org.broadinstitute.monster.dap.environment.WalkabilityTransformations
+import org.scalatest.OptionValues
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class WalkabilityTransformationsSpec extends AnyFlatSpec with Matchers with OptionValues {
```

Comment:
🔧 I think this test should also exercise the conditional codepaths around `wv_walkscore_date` and `wv_walkscore`